recce/cli.py

```diff
@@ -323,19 +325,29 @@ def server(host, port, lifetime, state_file=None, **kwargs):
         flag["single_env_onboarding"] = True
         flag["show_relaunch_hint"] = True
         target_path = kwargs.get("target_path")
-        target_base_path = kwargs.get("target_base_path")
         # Use the target path as the base path
         kwargs["target_base_path"] = target_path
 
         # Show warning message
         console.rule("Notice", style="orange3")
-        console.print("Recce is launching in single environment mode with limited functionality.")
         console.print(
-            "For full functionality, prepare a base set of dbt artifacts to compare against in "
-            f"'{target_base_path}'."
+            "Recce is ready to launch in Single Environment Mode with limited functionality."
+            "\n\n"
+            "Single Environment Mode allows you to explore your dbt project but won't show "
+            "data comparisons between environments. For full functionality, configure a "
+            "base environment."
+            "\n\n"
+            "To set up full environment comparison:"
+            "\n- Run `recce debug` for setup assistance"
+            "\n- Visit https://docs.datarecce.io/configure-diff/ for configuration guide"
+            "\n"
         )
-        console.print("https://docs.datarecce.io/get-started/#prepare-dbt-artifacts")
-        console.print()
+
+        single_env_flag = kwargs.get("single_env", False)
+        if not single_env_flag:
+            lanch_in_single_env = Confirm.ask("Launch in [bold]Single Environment Mode[/bold]?")
+            if not lanch_in_single_env:
+                os._exit(0)
```

Comment:
Hi Even,
Is there any specific reason to use `os._exit(0)`?
(just curious, because we used `exit(N)` in the codebase elsewhere)